truss/templates/shared/util.py

```diff
@@ -60,6 +63,19 @@ def all_processes_dead(procs: List[multiprocessing.Process]) -> bool:
     return True
 
 
+def kill_child_processes(parent_pid: int):
+    try:
+        parent = psutil.Process(parent_pid)
+    except psutil.NoSuchProcess:
+        return
+    children = parent.children(recursive=True)
+    for process in children:
+        process.terminate()
+    gone, alive = psutil.wait_procs(children, timeout=CHILD_PROCESS_WAIT_TIMEOUT)
```

Comment:
Good to add unit of this timeout to the name of the constant, I imagine it's seconds. We should increase this to 2 mins.